test/sanity/pep8/legacy-files.txt

```diff
@@ -1,3 +1,8 @@
+hacking/conf2yaml.py
+lib/ansible/constants.py
+lib/ansible/cli/config.py
+lib/ansible/config/data.py
+lib/ansible/config/manager.py
```

Comment:
I strongly recommend not doing this.  We're trying to clean it up anyway, no sense in just kicking the can down the road.